producer.go

```diff
@@ -810,7 +810,6 @@ func (p *producer) pollForSettingChanges(ctx context.Context, wg *sync.WaitGroup
 				// Don't log if this is part of a standard shutdown.
 				if !errors.Is(context.Cause(ctx), startstop.ErrStop) {
 					p.Logger.ErrorContext(ctx, p.Name+": Error fetching queue settings", slog.String("err", err.Error()))
-
```

Comment:
Not sure what happened here. My linter is suddenly picking these up, but it seems like it should've picked them up before. I left the changes in though because they look right and I'm sure the linter will want to put them in another PR if not this one.